pymaker/deployment.py

```diff
@@ -358,12 +393,18 @@ def approve_dai(self, usr: Address, **kwargs):
 
     def active_auctions(self) -> dict:
         flips = {}
+        clips = {}
         for collateral in self.collaterals.values():
-            # Each collateral has it's own flip contract; add auctions from each.
-            flips[collateral.ilk.name] = collateral.flipper.active_auctions()
+            # Each collateral has it's own liquidation contract; add auctions from each.
+            if collateral.flipper:
+                flips[collateral.ilk.name] = collateral.flipper.active_auctions()
+            elif collateral.clipper:
+                pass  # TODO: implement clipper.active_auctions()
```

Comment:
Is there a reason this can't be implemented?

Reply:
I forgot to wire that up after implementation; good catch.  Fixed and added tests in 4d68ac0fa276cc49cf7e82c5c6e4adb0ffdcc1e7.